FeBuddyLibrary/Helpers/FileHelpers.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FeBuddyLibrary.Helpers
+{
+    public class FileHelpers
+    {
+        private static string OutputPath { get; } = GlobalConfig.outputDirectory;
```

Comment:
Can't use properties or variables here because the class is technically not instantiated. Go back to using GlobalConfig.outputDirectory...